lib/helpers.js

```diff
@@ -848,35 +848,30 @@ function toAvdLocaleArgs (language, country) {
  * @returns {?string} The full path to the folder or `null` if the folder cannot be found
  */
 async function getAndroidPrefsRoot () {
-  // https://gerrit.pixelexperience.org/plugins/gitiles/development/+/5c11852110eeb03dc5a69111354b383f98d15336/tools/androidprefs/src/com/android/prefs/AndroidLocation.java
-  let location = null;
-  if (system.isWindows()) {
-    let localAppData = process.env.LOCALAPPDATA;
-    if (!localAppData) {
-      localAppData = process.env.USERPROFILE;
-      if (localAppData) {
-        localAppData = path.resolve(localAppData, 'Local Settings', 'Application Data');
-      }
-    }
-    if (localAppData) {
-      location = path.resolve(localAppData, 'Android');
-    }
-  } else {
-    const home = process.env.HOME;
+  let location = process.env.ANDROID_EMULATOR_HOME;
+  if (location && !await fs.exists(location)) {
```

Comment:
it also makes sense to check that this is a directory. Consider extracting such check itself into a separate function as it is used more than once